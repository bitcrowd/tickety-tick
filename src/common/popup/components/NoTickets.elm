module NoTickets exposing (..)

import Html exposing (..)
import Html.Attributes exposing (..)

import ExternalLink

import Messages exposing (Msg)


view : Html Msg
view =
  div [ class "text-xs-center m-b-2" ] 
    [ h5 [ class "m-t-2 m-b-2" ] [ text "No tickets found on this page." ]
    , h6 [] [ text "Did you select or open any tickets?" ]
    , p []
      [ text "Tickety-Tick currently supports"
      , br [] []
      , text "GitHub, Jira, Pivotal and Trello."
      ]
    , h6 [] [ text "Missing anything or found a bug?" ]
    , p [] [ ExternalLink.view "https://github.com/bitcrowd/tickety-tick/issues" "Report an issue" ]
    ]
