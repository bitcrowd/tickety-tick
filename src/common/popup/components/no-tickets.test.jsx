import React from 'react';
import { shallow } from 'enzyme';

import NoTickets from './no-tickets';

describe('about', () => {
  function render(/* overrides */) {
    return shallow(<NoTickets />);
  }

  it('renders', () => {
    expect(render({})).toMatchSnapshot();
  });
});
