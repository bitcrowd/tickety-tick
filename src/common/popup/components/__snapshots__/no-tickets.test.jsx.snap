// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about renders 1`] = `
<div
  className="text-center"
>
  <h5
    className="mt-3 mb-3"
  >
    No tickets found on this page.
  </h5>
  <h6>
    Did you select or open any tickets?
  </h6>
  <p>
    Tickety-Tick currently supports
    <br />
    GitHub, GitLab, Jira, Pivotal, Trello, and Ora.
  </p>
  <h6>
    Missing anything or found a bug?
  </h6>
  <p
    className="pb-1"
  >
    <ExternalLink
      href="https://github.com/bitcrowd/tickety-tick/issues"
    >
      Report an issue
    </ExternalLink>
  </p>
</div>
`;
